specification/source/cxf.rst

```diff
@@ -0,0 +1,290 @@
+                                                     OutputConnector, block.
+                                                     Parameter, 
+                                                     Constant
+   hasFmuPath                      ExtensionBlock    String           Used to include (local or over network)
+                                                                      path to a Functional Mockup Unit
+                                                                      implementation of an extension block.
+   isOfDataType                    InputConnector,   DataType         Used to define the data type for input 
+                                   OutputConnector,                   connectors, output connectors, parameters,
+                                   Parameter,                         and constants.
+                                   Constant
+   containsBlock                   Block             Block            Used in composite block to include other
+                                                                      blocks.
+   connectTo                       OutputConnector,  InputConnector,  Used to connect the output of one block
+                                   InputConnector    OutputConnector  to the input of a block. Only connectors
+                                                                      that carry the same data type can be
+                                                                      connected.
+   translationSoftware             Package, Block    String           Used to include the name of the software
+                                                                      used to CXF representation of the
+                                                                      sequence.
+   translationSoftwareVersion      Package, Block    String           Used to include the version of the
+                                                                      software used to CXF representation of
+                                                                      the sequence.		
+   =============================== ================= ================ =========================================
+
+All the ElementaryBlock within the standard have been 
+defined and included in ``CXF-Core.jsonld``. However, CXF 
+representation of elementary blocks does not contain 
+the implementation details of the blocks. 
+
+Generating CXF from an instance of a CDL class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A CXF representation of a sequence will be able to be 
+generated from a instance of a CDL class. When there 
+are instances of a CDL class within a Modelica or 
+another CDL class, if the instance has the CDL 
+annotation ``__cdl(export=true)``, the corresponding CDL 
+class shall be translated to CXF. If there is no 
+``export`` annotation attached to the instance, the 
+default value would be false and no CXF will be 
+generated for the class of that instance. 
```

Comment:
Change

>  If there is no 
``export`` annotation attached to the instance, the 
default value would be false and no CXF will be 
generated for the class of that instance. 

to
> Specifying the `export` annotation is optional, and if unspecified `export=false` is assumed.